src/HappyConnOpener.cc

```diff
@@ -437,17 +437,15 @@ HappyConnOpener::status() const
 
     SBufStream os(buf);
 
-    os.write(" [", 2);
     if (stopReason)
         os << "Stopped:" << stopReason;
     if (prime)
-        os << "prime:" << prime;
+        os << " prime:" << prime;
     if (spare)
-        os << "spare:" << spare;
+        os << " spare:" << spare;
     if (n_tries)
         os << " tries:" << n_tries;
     os << " dst:" << *destinations;
-    os << ' ' << id << ']';
```

Comment:
Why was the `id` field removed? Please note that AsyncJob::status() is used for more than just mgr:jobs.

Reply:
Same goes for the list brackets ('['...']') removal - they should be kept.

You have added the missing space, those also need comma's to make these a valid YAML/JSON set.

Reply:
> also need comma's to make these a valid YAML/JSON set.

While adding commas could fix _syntax_ in some cases, it would be semantically wrong -- the information in brackets is not an array or similar collection of same-type items.

As I wrote in another change request, we should split AsyncJob::status() method into two (instead of trying to make a single method do two different things well). IIRC, that was the plan when this report was recently added; we just did not want to mix two very different concerns (one of which would require a few noisy changes) into one PR.

Reply:
> Why was the `id` field removed? 

ID field has not been removed: it's the map key. The question should rather be why it used to be duplicated

> Please note that AsyncJob::status() is used for more than just mgr:jobs.

I've looked for more users where status() seems to provide a leading space, shifting responsibility to users. Hopefully I caught them all

Reply:
> Same goes for the list brackets ('['...']') removal - they should be kept.
> 
> You have added the missing space, those also need comma's to make these a valid YAML/JSON set.

Any objection to turning the brackets into curly? That would turn the field into a YAML inline map